tests/test_module_seletopclusts.py

```diff
@@ -0,0 +1,278 @@
+"""Test related to the seletopclusts module."""
+
+import os
+import pytest
+import pytest_mock  # noqa : F401
+import tempfile
+import math
+from pathlib import Path
+
+from . import golden_data
+
+from haddock.libs.libontology import PDBFile
+from haddock.modules.analysis.seletopclusts.seletopclusts import (
+    map_clusters_models,
+    select_top_clusts_models,
+    rank_clust_order,
+    size_clust_order,
+    sort_models,
+    write_selected_models,
+    )
+
+
+NB_CLUSTERS = 3
+MDL_RANK_INCREMENT = 5
```

Comment:
what do these two variables represent?

Reply:
A way to standardize the counting in generation of test datasets and later tests.